### DISCUSSION TITLE
Branches is missing from my repos settings page. Why?

### DISCUSSION THREAD

**Discussion by kga245** [community/community#84994] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am banging my head against a wall trying to figure out why one (and only one) of my repos does not have a "Branches" section under "Code and automation" section of the "Settings" page. 

Here is the documentation: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule

Below you can see a screenshot showing two of my repos by comparison. One is just missing the section altogether and I have no idea why. 

<img width="2873" alt="Mouse_Highlight_Overlay_and_General_and_General" src="https://github.com/community/community/assets/74297/7642e88e-21d2-4f1f-b1a9-0f34f9e0f61b">

**Comment by kga245** [comment, marked as answer]
After more hair pulling I found a key difference in the two repos. Branch protection rules were disabled at the repo settings level found here: https://github.com/[organization]/[repository]/settings
<img width="880" alt="Mouse_Highlight_Overlay" src="https://github.com/community/community/assets/74297/7266898e-7fef-4c08-8304-37cad571b94e">

**Reply by doughlass** [reply]
Could have been done with finding this 2 hours ago 👍
